src/base/rss/rssfeed.cpp

```diff
@@ -86,6 +86,10 @@ Feed::Feed(const QString &url, Manager *manager)
     refresh();
 }
 
+Feed::Feed(const QString &url)
+    : Feed(url, nullptr)
+{}
```

Comment:
>or only apply when on a single line?

Yes.
We force multiline for constructors with initialization list and therefore we split its body braces.